src/components/modules/paste.ts

```diff
@@ -324,13 +335,31 @@ export default class Paste extends Module {
 
         return;
       }
-
-      this.toolsTags[tag.toUpperCase()] = {
-        tool,
-      };
+      /** Sanitization configuration is string */
+      if (_.isString(tag)) {
```

Comment:
Maybe we should rename the variable? Something like `tagOrSanitizeConfig`, because it's confusing rn — comment says about config but variable name is `tag`